recipes-multimedia/imx-sw-pdm/imx-sw-pdm_1.0.1.bb

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 NXP Semiconductors
+
+DESCRIPTION = "NXP PDM to PCM Software Decimation SIMD Library"
+LICENSE = "Proprietary"
```

Comment:
The current description would fit for a summary. Please add a description to this.

Reply:
Please add a more detailed description as the current one does not explain what it provides.

Reply:
@aalonso, can you help me here?